packages/runtime/src/utils/share.ts

```diff
@@ -151,6 +151,7 @@ export function getRegisteredShare(
   instanceName: string,
   pkgName: string,
   shareInfo: ShareInfos[keyof ShareInfos],
+  resolveShareHook: any,
```

Comment:
Can you help to add the hook types @zhoushaw ?

Reply:
I think set the resolveShareHook as partial option is better, what do you think ? @ScriptedAlchemy